modules/install-sno-about-single-node-openshift.adoc

```diff
@@ -0,0 +1,10 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-preparing-to-install-sno.adoc
+
+[id="install-sno-about-single-node-openshift_{context}"]
+= About single node OpenShift
+
+Single node OpenShift installs a self-contained {product-title} instance on a single bare metal node. The primary use case is for edge computing workloads, including small physical footprints, intermittent connectivity, portable clouds, and 5G radio access networks (RAN) close to a base station. A single node {product-title} installation supports autonomous workloads.
+
+single node OpenShift removes some requirements associated with deploying larger clusters. The major trade off with single node OpenShift installations is the lack of high availability. Currently, Red Hat does not support upgrading a single node OpenShift installation. Upgrades and some configuration changes require re-installation.
```

Comment:
"Installing {product-title} on a single node removes..."

"The major trade off with an installation on a single node is the lack of high availability."

I'll stop with the suggestions at this point, but please do use the words "single node OpenShift" together and do not use "OpenShift" alone, use {product-title}.